From bf4d18af7316eb6bf29a53d39daea1d2b526254e Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 12 Sep 2019 11:31:40 +1000
Subject: [PATCH] meson.build: drop yaml from the required python module list

This isn't required for the flatpak, the yaml module is only needed in some of
the debugging/recovery tools but not in Tuhi itself.
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 3b2db56..c389d65 100644
--- a/meson.build
+++ b/meson.build
@@ -35,7 +35,6 @@ python_modules = [
     'svgwrite',
     'xdg',
     'gi',
-    'yaml',
 ]
 if meson.version().version_compare('>=0.51')
     py3 = pymod.find_installation(modules: python_modules)
-- 
2.21.0

